src/bin/ntfs2mimir.rs

```diff
@@ -116,11 +116,11 @@ fn to_mimir(
             name: navitia.physical_modes[pm_idx].name.clone(),
         })
         .collect();
-    let comments = navitia
-        .comments
-        .iter_from(&stop_area.comment_links)
-        .map(|comment| mimir::Comment {
-            name: comment.name.clone(),
+    let comments = stop_area
+        .comment_links
+        .iter()
+        .map(|com_id| mimir::Comment {
+            name: navitia.comments.get(com_id).unwrap().name.clone(),
```

Comment:
normally `transit_model` ensures that objects do not reference other non-existent objects.
So the `unwrap` is safe here. but who knows?

If you want the code to be safer you can omit the comments that don't exist (and log)
```suggestion
    let comments = stop_area
        .comment_links
        .iter()
        .filter_map(|com_id| navitia.comments.get(com_id))
        .map(|c| mimir::Comment {
            name: c.name.clone(),
```

Reply:
Great. I will do that. Although I'm not sure of the consequences of having empty comments.... But no real programmer should be worried by a lack of comment ;-)